### DISCUSSION TITLE
python开发插件

### DISCUSSION THREAD

**Discussion by dao-zero** [lobehub/lobehub#1253] [Development Question]
我想使用python开发一款插件，那么网关要怎么弄呀，有py版的lobehub吗，没有的话，这个网关具体做了什么可以告知一下吗

**Comment by arvinxx** [comment]
我们会在后续提供 python 的插件开发模板。但目前暂时没有python 网关，所以开发可能暂时会有点问题，我们会在后续提供出来插件网关的 cli，这样就可以直接在本地启动起来，能做到与语言无关。
